### PR TITLE
Migrating string falses to boolean

### PR DESCRIPTION
**Pull request by Zenen-freshworks**
Actually went really smoothly once I stopped overcomplicating it!

### CODE REVIEW

**Review by mauriciofreshw — APPROVED** [review]

**Review by mauriciofreshw — DISMISSED** [review]

File: cypress/integration/participantView.spec.js [hunk 1]
```diff
@@ -11,8 +11,16 @@ describe("Participant View", () => {
     cy.get('ul.MuiMenu-list[role=listbox] > li').should('have.length', 3);
 
     // Testing for freshness
-    cy.contains('a day ago').should('exist');
-    cy.contains('14 days ago').should('exist');
+
+    cy.get('body').then(($body) => {
+      if (Cypress.$('body:contains("a day ago")').length !== 0) {
```
Comment (Zenen-freshworks) [inline cypress/integration/participantView.spec.js:16]: The response of moment's .fromNow() function fluctuates by a day based on when the test is run. This should cover both cases. I don't want to spend much more time on writing this test but if it seems important to figure out exactly where the discrepancy is coming from then I'll dig deeper.

Comment (ashtonmeuser) [reply cypress/integration/participantView.spec.js:16]: Great [lampshading](https://tvtropes.org/pmwiki/pmwiki.php/Main/LampshadeHanging).

**Review by ashtonmeuser — APPROVED** [review]

File: cypress/integration/participantView.spec.js (hunk 1 above)
Comment (fwkendall) [reply cypress/integration/participantView.spec.js:16]: I bet this is down to the time of the server used by GHA when running the tests (or something like that)

File: server/server.js [hunk 2]
```diff
@@ -237,7 +237,7 @@ app.post(`${apiBaseUrl}/new-hired-participant`,
       [participantInfo.preferredLocation] = user.regions;
       participantInfo.crcClear = 'yes';
       participantInfo.interested = 'yes';
-      participantInfo.callbackStatus = 'false';
+      participantInfo.callbackStatus = false;
```
Comment (fwkendall) [inline server/server.js:240]: Great catch - I wouldn't have been able to see this problem in the UI due to the participant already being hired 😄

**Review by fwkendall — APPROVED** [review]

**Pull request merged**